security/pfSense-pkg-snort/files/usr/local/pkg/snort/snort_migrate_config.php

```diff
@@ -219,10 +203,10 @@
 /* large numbers of pfSense users hitting Snort.org at    */
 /* the same minute past the hour for rules updates.       */
 /**********************************************************/
-if (empty($config['installedpackages']['snortglobal']['rule_update_starttime']) || 
-	  $config['installedpackages']['snortglobal']['rule_update_starttime'] == '00:05' || 
-	  strlen($config['installedpackages']['snortglobal']['rule_update_starttime']) < 5 ) {
-	$config['installedpackages']['snortglobal']['rule_update_starttime'] = "00:" . str_pad(strval(random_int(0,59)), 2, "00", STR_PAD_LEFT);
+if (empty(config_get_path('installedpackages/snortglobal/rule_update_starttime')) ||
+	  config_get_path('installedpackages/snortglobal/rule_update_starttime') == '00:05' ||
```

Comment:
```suggestion
if (config_get_path('installedpackages/snortglobal/rule_update_starttime', '00:05') == '00:05' ||
```

Reply:
Same as previous. Made the suggested change.